apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -210,6 +213,25 @@ export class ChangeAction extends BaseRushAction {
     return false;
   }
 
+  private _findRushPathDiff(changedFolders: Array<string | undefined>,
+    project: RushConfigurationProject): string {
+      const normalizedFolder: string = project.projectRelativeFolder;
+      let rushPathDiff: string = '';
+      for (const folder of changedFolders) {
+        if (folder !== undefined) {
+          const splitted: Array<string | undefined> = folder.split('/');
+          if (splitted[0] !== normalizedFolder) {
+            // if rush.json is not under the root directory
+            const changeProjectIndex: number = splitted.indexOf(normalizedFolder);
+            for (let i: number = 0; i < changeProjectIndex; i++) {
+              rushPathDiff = rushPathDiff + splitted[i] + '/';
```

Comment:
Use `path.join` here too.

```suggestion
              rushPathDiff = path.join(rushPathDiff, splitted[i]);
```